onnx_tf/handlers/backend/non_max_suppression.py

```diff
@@ -16,15 +16,24 @@ def _common(cls, node, **kwargs):
     # max_output_boxes for tf.image.non_max_suppression must be in tf.int32
     # therefore need to cast this input to tf.int32
     max_output_boxes_per_class = tf.cast(
-        tensor_dict['max_output_boxes_per_class'], tf.
-        int32) if 'max_output_boxes_per_class' in tensor_dict else tf.constant(
-            [tf.cast(boxes.shape[1], tf.int32)], tf.int32)
-    iou_threshold = tensor_dict[
-        'iou_threshold'] if 'iou_threshold' in tensor_dict else tf.constant(
-            [0.5], tf.float32)
-    score_threshold = tensor_dict[
-        'score_threshold'] if 'score_threshold' in tensor_dict else tf.constant(
-            [float('-inf')], tf.float32)
+        tensor_dict[node.inputs[2]],
+        tf.int32) if (len(node.inputs) > 2 and
+                      node.inputs[2] != "") else tf.constant(0, tf.int32)
```

Comment:
according to spec, `max_output_boxes_per_class` should be `int64`

Reply:
@pluradj max_output_boxes for tf.image.non_max_suppression must be in tf.int32

Reply:
Ha, you're right! I didn't see the code comment above the diff. Perhaps it should be noted as a restriction with `@ps_description`.